src/Console/AlgoliaCommand.php

```diff
@@ -2,25 +2,29 @@
 
 namespace Algolia\Settings\Console;
 
+use Algolia\Settings\Services\Resource as ResourceService;
```

Comment:
I hope we can come up with a more explicite name, but I don't have much inspiration. "Resource" is how we call `settings`, `rules`, `synonyms` at Algolia but it might be not very clear for everyone. Maybe `Algolia\Settings\IndexResource` ?

Reply:
Ugh, naming... Ok, this package is small enough not to need a separate namespace for the service... :-) 

I'm really at a loss for a specific name too. Possible alternatives to `IndexResource`:
- `IndexManager`
- `IndexReader`
- `IndexRepository`

Also, maybe instead of `Index` use `Settings`?

Reply:
I love `Algolia\Settings\IndexRepository` 👍 I think it makes a lot of sense.